database/sql/V068__executed_protocol_fees.sql

```diff
@@ -0,0 +1,8 @@
+-- Create a new type to store the executed protocol fee and the token the fee is paid in.
+CREATE TYPE FeeAsset AS (
+    amount numeric(78,0),
+    token bytea
+);
```

Comment:
I don't think we should be using advanced postgres features (in case we ever want to change the underlying database). What's the advantage of defining a struct here, which we haven't done for any of the structs?

Reply:
now we have two plain vectors